template_feed/Microsoft.DotNet.Common.ProjectTemplates.5.0/content/ClassLibrary-FSharp/Company.ClassLibrary1.fsproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.ClassLibrary1</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <WarnOn>3390;$(WarnOn)</WarnOn>
```

Comment:
I forgot if this change was meant for 5.0.300 or 6.0... Do you remember?

Reply:
5.0.300